crml/cennzx-spot/src/lib.rs

```diff
@@ -675,7 +677,11 @@ impl<T: Trait> Module<T> {
 		let denominator = output_reserve - output_amount;
 		let output: T::Balance = numerator / denominator + One::one();
 
-		(FeeRate::one() + fee_rate) * output
+		let output: T::AsBalance = output.into();
+		let output: u128 = output.as_();
+		let result = (FeeRate::one() + fee_rate) * output;
+		let result: T::AsBalance = As::sa(result);
+		Into::<T::Balance>::into(result)
```

Comment:
is this like a safe mul? not understand the hardcode u128 part.

Reply:
This is same behavior as the old code, but because we are dropping `As<u128>` from `GA::Balance`, we need some workarounds here.
The proper solution is make FeeRate not needing `As<u128>` anymore but it is too much for this PR.